src/main.tsx

```diff
@@ -3,6 +3,7 @@ import React from "react";
 import ReactDOM from "react-dom/client";
 import { BrowserRouter, Route, Routes } from "react-router-dom";
 import "./global/default.css";
+import WelcomeScreeen from "./components/welcome.screen/welcome.screen";
```

Comment:
there is a typo: 
should be import WelcomeScreen instead of "WelcomeScreeen" from "./components/welcome.screen/welcome.screen";

Reply:
Thank you for your comment. Fixed.